isis/src/base/objs/CameraFactory/CameraFactory.truth

```diff
@@ -3,29 +3,29 @@ Testing missing Instrument Group ...
 Version: **ERROR** Unable to locate latest camera model version number from group [Instrument].
 **ERROR** Unable to find PVL group [Instrument].
 
-**ERROR** Unable to initialize camera model from group [Instrument].
+**ERROR** Unable to initialize camera model in Camera Factory.
```

Comment:
If the file and line numbers are turned on, this change to the error text might be redundant.

I understand why it was done.